src/models/applicant.ts

```diff
@@ -0,0 +1,51 @@
+import {Schema, SchemaType, SchemaTypes} from "mongoose";
+
+const applicantSchema = new Schema(
+    {
+        firstName: {
+            type: String,
+            trim: true,
+            required: true,
+        },
+        lastName: {
+            type: String,
+            trim: true,
+            required: true,
+        },
+        role: {
+            type: Number,
+            enum: [0, 1],
+            default: 0,
+            required: true,
+        },
```

Comment:
I think in enumerators, you can just specify them as strings within this field https://intellipaat.com/community/34313/how-to-create-and-use-enum-in-mongoose

You wont need the enum Role at the bottom in this case then.